File: server/app/Models/PaymentGateway.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PaymentGateway extends Model
{

    protected $table = 'PaymentGateways';
    public $timestamps = false;

    protected $fillable = [
        'id',
        'paymentGatewayName',
        'cashInNumber',
        'personalNumber',
        'agentNumber',
        'ip',
        'modifiedBy',
        'createdAt',
    ];
}

File: client/lib/models/PaymentGateway.dart
class PaymentGateway{
  int id;
  String paymentGatewayName;
  String cashInNumber;
  String personalNumber;
  String agentNumber;

  PaymentGateway({
    this.id,
    this.paymentGatewayName,
    this.personalNumber,
    this.agentNumber,
    this.cashInNumber
  });
}